README.md

```diff
@@ -43,6 +43,10 @@ var pool = new Pool({
     idleTimeoutMillis : 30000,
     // Delay in milliseconds after which pending acquire request in the pool will be rejected.
     acquireTimeoutMillis: 30000,
+    // optional. if you set this, the resource will be destroyed and replaced after it has been used
+    // `maxUses` number of times, which can help with re-balancing when pool members are added after
+    // the process has started and already filled the pool with healthy connections.
+    maxUses  : 1000,
```

Comment:
Just a question (not a problem with the PR): is this value of `1000` a real-life example? If yes, why is 1000 a good value?

Reply:
A real-life example will probably differ significantly depending on volume...

Would it be less confusing to provide some example math for determining a reasonable number?

Reply:
I don't have experience with this these connection pools... It would be excellent if you could make the example closer to a real-life situation, in whatever way you think is good - unless it is too complicated and gets off-topic.

Reply:
I added a discussion section toward the bottom of the readme that spells out an example similar to the one we are currently encountering (Amazon Aurora, by the way).